src/app/form-entry/models/question-base.ts

```diff
@@ -0,0 +1,23 @@
+import { Validation } from './validation';

```

Comment:
see comments on model in: https://github.com/AMPATH/ng2-opemmrs-formentry/pull/4/files